src/roadrunner-symfony-nyholm/src/Runner.php

```diff
@@ -73,6 +52,7 @@ public function run(): int
             } catch (\Throwable $e) {
                 $worker->getWorker()->error((string) $e);
             } finally {
+                // TODO move this cleanup also to Symfony
```

Comment:
The following part is also required in all cases, but not yet sure where to put it into symfony?

```php
                if (PHP_SESSION_ACTIVE === session_status()) {
                    session_abort();
                }
                // reset all session variables to initialize state
                $_SESSION = [];
session_id(''); // in this case session_start() will generate us a new session_id()
```